app/src/main/java/com/mathroule/demoapp/MainActivity.java

```diff
@@ -28,6 +30,24 @@ public void onClick(View view) {
                         .setAction("Action", null).show();
             }
         });
+
+        try {
+            String text = Helper.getRandomButtonLabel();
+            ((Button) findViewById(R.id.buton)).setText(text);
+
+            if (text == "something") {
+                ((Button) findViewById(R.id.buton)).setText("something");
+            } else           if (text == "something-else") {
+                ((Button) findViewById(R.id.buton)).setText("something-else");
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mathroule/demo-app/blob/c0ca0570c1175a60455e68081fbe6ddf3053938f/app/src/main/java/com/mathroule/demoapp/MainActivity.java#L41"></span>String literal in <code>setText</code> can not be translated. Use Android resources instead.</td>
    </tr>
  </tbody>
</table>